docs/storefront/headless/routes.mdx

```diff
@@ -0,0 +1,269 @@
+  <Tab>
+    ```graphql filename="Query product details" showLineNumbers copy
+
+      query {
+       site {
+         route(path: "/product_1") {
+           redirect {__typename to {__typename ...on ProductRedirect{entityId path}} toUrl} 
+           node {
+             id
+             ... on Product {
+               name
+               }
+              }
+             }
+           }
+         }
+    ```
+ </Tab>
+<Tab>
+```json filename="Example response" showLineNumbers copy
+
+     {
+  "data": {
+    "site": {
+      "route": {
+        "redirect": {
+          "__typename": "Redirect",
+          "to": {
+            "__typename": "ProductRedirect",
+            "entityId": 123,
+            "path": "/t-shirt/"
+          },
+          "toUrl": "https://{example-store}.mybigcommerce.com/t-shirt/"
+        },
+        "node": null
+      }
+    }
+  }
+}
+```
+</Tab>
+</Tabs>
+
+
+### Query: Fetching route details
+
+This example retrieves details about an object (e.g., Product, Brand, or Category) based on a given URL path.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```graphql filename="Query route details" showLineNumbers copy
+
+     query LookUpUrl(
+       $urlPath: String!
+       # Use GraphQL Query Variables to provide a path
+     ) {
+       site {
+         route(path: $urlPath) {
+           node {
+             __typename
+             id
+             # A different response is returned based on which type of object was matched
+             ... on Category {
+               name
+               description
+             }
+             ... on Brand {
+               name
+               defaultImage {
+                 url(width: 200)
+               }
+             }
+             ... on Product {
+               name
+               description
+               images {
+                 edges {
+                   node {
+                     url(width: 500, height: 500)
+                   }
+                 }
+               }
+             }
+             ... on Redirect {
+               toUrl
+             }
+           }
+         }
+       }
+     }
+```
+```json filename="GraphQL variables" showLineNumbers copy 
+    {
+      "urlPath": "/Blouse/"
+    }
+```
+  </Tab>
+  <Tab>
+    ```json filename="Example response" showLineNumbers copy
+    {
+  "data": {
+    "site": {
+      "route": {
+        "node": {
+          "__typename": "Product",
+          "id": "UHJvZHVjdDoxNDE=",
+          "name": "Blouse",
+          "description": "<p>Blouse</p>\n<p>#blouse</p>",
+          "images": {
+            "edges": [
+              {
+                "node": {
+                  "url": "https://cdn11.bigcommerce.com/s-w0astee9jo/images/stencil/500x500/products/141/396/blouse__67850.1689699580.jpg"
+                }
+              }
+            ]
+          }
+        }
+      }
+    }
+  }
+}
+   ```
+</Tab>
+</Tabs>
+
+### Query: Using `IGNORE` redirect behavior
```

Comment:
This example is basically the same as your first one that is called "Query: Fetching a product".

I like this example and description better than that one and think you should probably just remove the first example altogether. Making "Query: Fetching route details" your first example makes more sense to me.